### PR TITLE
Jared/readme sub module

### PR DESCRIPTION
**Pull request by jared-2016**
@ldanzinger Ready for review. Thank you!

Let me know what you think on these changes referencing the toolkit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ ArcGIS Runtime SDK 100.10 for Qt sample applications.  The repo contains [Qt](ht
 # Prerequisites
 * Qt SDK 5.15 or higher (Open source or commercial)
```
Comment (ldanzinger) [inline README.md:5]: Just noticed - this should be 5.15.1

File: README.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ ArcGIS Runtime SDK 100.10 for Qt sample applications.  The repo contains [Qt](ht
...
 * Qt Creator
+* [ArcGIS Runtime Toolkit - Qt](https://github.com/Esri/arcgis-runtime-toolkit-qt) is required as some samples utilize toolkit components. For more information on the toolkit please visit, [Building a better toolkit.](https://community.esri.com/community/developers/native-app-developers/arcgis-runtime-sdk-for-qt/blog/2020/08/28/building-a-better-toolkit)
```
Comment (ldanzinger) [inline README.md:7]: i'd leave out the blog link - that's more of a background detail that isn't required for basic setup instructions of the sample repo

```suggestion
* [ArcGIS Runtime Toolkit - Qt](https://github.com/Esri/arcgis-runtime-toolkit-qt) is required as some samples utilize toolkit components. 
```

File: README.md [hunk 3]
```diff
@@ -21,6 +22,21 @@ Open your terminal, navigate to your working directory, use ```git clone``` to g
...
+```
+# Change directory to the clone of the samples repository
+# Clone the toolkit repository into the current directory in terminal
+$ cd /arcgis-runtime-samples-qt
+$ git clone https://github.com/Esri/arcgis-runtime-toolkit-qt.git
+```
+
+Cloning the toolkit in this location will allow for the samples to automatically pick it up. If you wish to place the toolkit in another location, you will need to update the samples project file accordingly to locate the necessary .pri file.
+
+* Note - if you want to contribute to the toolkit repository as well, you will need to follow the fork step, [Fork the repo](#fork-the-repo), from above but for the toolkit instead.
```
Comment (ldanzinger) [inline README.md:38]: > [Fork the repo](#fork-the-repo), from above but for the toolkit instead.

I get what you mean, but this is a bit clunky. I'd maybe just leave this detail out - most people won't want to contribute back to the project. The toolkit repo has instructions on how to do it, so I think we can leave it out from here.

**Review by ldanzinger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jared-2016) [reply README.md]: fixed

**Comment by jared-2016** [comment]
Should be good to go.

**Review by ldanzinger — APPROVED** [review]
LGTM

**Comment by jared-2016** [comment]
updates samples readme to give instructions on the proper location for the toolkit as it's needed for individual samples.

**Pull request merged**